content/docs/studio/user-guide/model-registry/version-models.md

```diff
@@ -0,0 +1,45 @@
+# Version models
+
+You can register new versions of the registered models by specifying the Git
+commit which corresponds to the new version.
+
+To register a new version of a model, Iterative Studio uses [GTO] to create an
+[annotated Git tag][git tag] with the specified version number. Refer the [GTO
+docs][gto] to see the exact format of the Git tag.
+
+You can register versions using the [GTO] command line interface (CLI). To
+register versions using Iterative Studio, watch this tutorial video or read on
+below:
+
+**_TODO: Replace the below with the tutorial video on registering a new version
+of a model_**
+
+https://www.youtube.com/watch?v=hKf4twg832g
+
+1. On the model details page, click on `Register new version`. The registration
```

Comment:
Nit: If we are writing down a flow for registering a new version, does it make more sense do this from the model dashboard? 
1. We already receive updates for these actions without reloading the page.
2. Once the action completes, clicking the row in the Model Table takes you to your new version straight away. If you reload a model detail page, you're still on the version you were looking at when you registered a new version.

Reply:
Changed.